### DISCUSSION TITLE
jolt physics

### DISCUSSION THREAD

**Discussion by WOLFxxxxxx** [luis605/Lit-Engine-OLD#40] [Ideas]
I prefer to use a jolt motor instead of a bullet.
https://github.com/rodneylab/jolt-raylib-hello-world

![68747470733a2f2f6a726f7577652e6e6c2f6a6f6c742f486f72697a6f6e5f466f7262696464656e5f576573742e706e67](https://github.com/user-attachments/assets/aabbf177-6c73-4037-90ab-137a78a76d4b)

**Comment by luis605** [comment]
Why should bullet3 get replaced by bolt?

**Comment by WOLFxxxxxx** [comment]
![Screenshot_٢٠٢٤-١٠-١٢-٢١-٢٨-٢٩-٢٠٠_com google android youtube](https://github.com/user-attachments/assets/de22b863-2221-4e69-bf9d-00c439064b3f)
![Screenshot_٢٠٢٤-١٠-١٢-٢١-٢٨-٥٨-٢٤٧_com google android youtube](https://github.com/user-attachments/assets/24e55228-66e9-4d68-b8ad-d073d2ae100d)

Bullet physics is poorly done

**Comment by luis605** [comment]
Sorry, no plans for now.
